mqtt/mqtt.js

```diff
@@ -109,6 +110,18 @@ aedes.authorizePublish = function (client, packet, publish) {
             token: client.password,
             chanID: channelId
         },
+        // Parse unlimited subtopics
+        baseLength = 3, // First few elements which represents the base part of topic.
+        elements = packet.topic.split('/').slice(baseLength),
+        baseTopic = 'channel.' + channelId;
+     // Remove empty elements
```

Comment:
PLease align the comment.

Reply:
@drasko Its aligned with next line. Check the code again pls.